packages/mdc-select/README.md

```diff
@@ -1,9 +1,24 @@
-# MDC Select
+<!--docs:
+title: "Select"
+layout: detail
+section: components
+iconId: menu
+-->
 
-MDC Select provides material design single-option and multi-option select menus. It functions analogously to the
+# Select
+
+MDC Select provides Material Design single-option and multi-option select menus. It functions analogously to the
 browser's native `<select>` element, and includes a gracefully degraded version that can be used
 in conjunction with the browser's native element. Both are fully accessible, and fully RTL-aware.
 
+## Design & API Documentation
+
+<ul class="icon-list">
+  <li class="icon-list-item icon-list-item--spec">
+    <a href="https://material.io/guidelines/components/menus.html">Menus</a>
```

Comment:
Honestly, I find both articles so bizarrely organized. It doesn't look like I can link directly to anything that's about select controls. Should I include the link at all?